packages/core/src/Account.ts

```diff
@@ -296,7 +296,11 @@ export class Account<T = any> extends EventEmitter {
       //call /access endpoint with client_id after client initialisation
       await this.apiClient.transport.http.associateClientWithSession(localClient.id);
 
-      if (this.cryptoProtocolConfig?.mls && this.backendFeatures.supportsMLS) {
+      await this.service.proteus.init();
+      if (this.backendFeatures.supportsMLS && this.cryptoProtocolConfig?.mls) {
+        const {userId, domain} = this.apiClient.context;
+        const clientId = constructFullyQualifiedClientId(userId, localClient.id, domain || '');
```

Comment:
not sure about this `domain || ''`, what should we do if domain is not defined? throw?

Reply:
The case of the empty domain should never happen on any of our env. I think it's fine to keep it like this for now